packages/api/src/rx/index.ts

```diff
@@ -190,7 +190,7 @@ export default class ApiRx extends ApiBase<OnCall> implements ApiRxInterface {
     return this._isReady;
   }
 
-  protected onCall (method: OnCallFunction<Observable<Codec | undefined | null>>, params: Array<any>): OnCall {
+  protected onCall (method: OnCallFunction<RxResult, RxResult>, params: Array<CodecArg> = []): RxResult {
```

Comment:
Since we're assuming everywhere that there's only one param, for consistency, we could only put `param: CodecArg` here?

Reply:
onCall is used for anything, i.e. only storage has 1 param, derive, rpc, etc. have multiples. (Still need to deal with that is some way in those typings...)

Reply:
Ah, correct.

Reply:
... in the future, will probably do the `arg: CodecArg, arg: CodecArg, ...` dance everywhere else, here we can forgo it since it is internal. ()